### PR TITLE
Add logger

### PR DESCRIPTION
**Pull request by Sergey113222**

### CODE REVIEW

File: src/main/resources/application.properties [hunk 1]
```diff
@@ -8,7 +8,16 @@ themoviedb.ord.scheme=https
 themoviedb.ord.host=api.themoviedb.org
 spring.jpa.show-sql=true
 spring.jpa.properties.hibernate.dialect=org.hibernate.dialect.MySQL5Dialect
+spring.jpa.properties.hibernate.format_sql=true
```
Comment (RomanKashenok) [inline src/main/resources/application.properties]: keep it false better. Or use it depends on profile (prod-dev-test and ctr.)

File: src/main/resources/application.properties [hunk 2]
```diff
@@ -8,7 +8,16 @@ themoviedb.ord.scheme=https
...
 spring.jpa.hibernate.ddl-auto=none
 
+logging.file.path=logs
+logging.level.ROOT=INFO
+logging.level.info.movietrash=DEBUG
+logging.level.org.springframework.cache=TRACE
+logging.level.org.hibernate.type.descriptor.sql.BasicBinder=TRACE
+logging.level.org.hibernate.SQL=TRACE
+logging.level.org.springframework.web.client.RestTemplate=TRACE
```
Comment (RomanKashenok) [inline src/main/resources/application.properties]: have no idea why do you need debug and trace here (it'll pring all app and system related information - your error log will be owerflowed)

**Review by RomanKashenok — APPROVED** [review]
I'd advice better to use logging level INFO+

File: src/main/resources/application.properties (hunk 2 above)
Comment (Sergey113222) [reply src/main/resources/application.properties]: deleted

File: src/main/resources/application.properties (hunk 1 above)
Comment (Sergey113222) [reply src/main/resources/application.properties]: changed to false

**Pull request merged**